quay-enterprise/clair.md

```diff
@@ -212,6 +212,16 @@ To configure Clair to run under TLS, a few additional steps are required:
 2. Place these files as `clair.crt` and `clair.key` in your Clair configuration directory
 3. Uncomment the `key_file` and `crt_file` lines under `verifier_proxies` in your Clair `config.yaml`
 
+### Configuring trust to self-signed SSL
+
+To configure Clair with a Quay Registry that is configured with self-signed SSL certificates follow steps below:
+
+1. Rename the Quay Registry trusted certificate bundle to ca.crt 
```

Comment:
Customer needs to bring the file however they sign the Certificate. If they are self-signing with something like openssl, they would follow something like; https://tectonic.com/quay-enterprise/docs/latest/quay-ssl.html . If they use something like Active Directory Certificate Authority, they would follow Microsoft Documentation to do so.